dfsclient/src/test/java/org/hpccsystems/dfs/client/DFSReadWriteTest.java

```diff
@@ -88,6 +88,93 @@ public void readWithForcedTimeoutTest() throws Exception
         assertEquals("Not all records loaded",expectedCounts[0], records.size());
     }
 
+    @Test
+    public void allCharsNullTest() throws Exception
```

Comment:
minor but name implies this is testing a value w/ all nulls, I think nullCharTests might be a little more accurate?